src/pages/settings/Payments/PaymentsPage/index.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import {View} from 'react-native';
+import PaymentMethodList from '../PaymentMethodList';
+import ROUTES from '../../../../ROUTES';
+import HeaderWithCloseButton from '../../../../components/HeaderWithCloseButton';
+import ScreenWrapper from '../../../../components/ScreenWrapper';
+import Navigation from '../../../../libs/Navigation/Navigation';
+import styles from '../../../../styles/styles';
+import withLocalize, {withLocalizePropTypes} from '../../../../components/withLocalize';
+import compose from '../../../../libs/compose';
+import Button from '../../../../components/Button';
+import KeyboardAvoidingView from '../../../../components/KeyboardAvoidingView';
+import FixedFooter from '../../../../components/FixedFooter';
+import getPaymentMethods from '../../../../libs/actions/PaymentMethods';
+import Popover from '../../../../components/Popover';
+import {PayPal} from '../../../../components/Icon/Expensicons';
+import MenuItem from '../../../../components/MenuItem';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+class PaymentsPage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.PAYPAL = 'payPalMe';
+
+        this.state = {
+            showAddPaymentMenu: false,
+            anchorPositionTop: 0,
+            anchorPositionLeft: 0,
+        };
+
+        this.paymentMethodPressed = this.paymentMethodPressed.bind(this);
+        this.addPaymentMethodTypePressed = this.addPaymentMethodTypePressed.bind(this);
+        this.hideAddPaymentMenu = this.hideAddPaymentMenu.bind(this);
+    }
+
+    componentDidMount() {
+        getPaymentMethods();
+    }
+
+    paymentMethodPressed(nativeEvent, account) {
+        if (account) {
+            // Show the make default/delete popover
+        } else {
+            const position = nativeEvent.currentTarget.getBoundingClientRect();
```

Comment:
Does react native's onPress event not return position information?

Reply:
Chatted about this with Marc 1:1 but while `onPress` returns position information, we don't actually want the location of the press, we want the location of the element itself so we can line the dialog up